src/components/EditCoursePage/index.jsx

```diff
@@ -747,6 +764,7 @@ class EditCoursePage extends React.Component {
     const numberKey = key_for_reruns || key;
     const number = numberKey && getCourseNumber(numberKey);
     const entitlement = entitlements && entitlements[0];
+    const addMetadataOptions = additionalMetadataOptions.data ? additionalMetadataOptions.data : [];
```

Comment:
Left a comment on backend PR, lets add AM options in Course API as opposed to adding a new endpoint for that.